src/chrome/content/rules/Elsevier.xml

```diff
@@ -17,131 +17,54 @@
 	CDN buckets:
 
 		- elseviercdn.squizltd.netdna-cdn.com
+		- cdn.elsevier.com
 
-			- cdn.elsevier.com
-			- -ssl doesn't exist
 
+	Invalid certificate:
+		www.ethics.elservier.com
+		res.journals.elsevier.com
+		cdn.elservier.com
+		developers.elsevier.com
+		www.ees.elsevier.com
 
-	Nonfunctional hosts in *elsevier.com:
-
-		- www.ethics *
-		- res.journals
-		- reprints *
-
-	* Refused
-
-
-	Problematic hosts in *elsevier.com:
-
-		- cdn		(404, mismatched, CN: *.netdna-ssl.com)
-		- dev ²
-		- developers ³
-		- (www.)?ees ⁴
-		- support	(Redirect destination mismatched)
-
-	² Mixed css
-	³ Mismatched
-	⁴ Expired
-
+	Refused:
+		reprints.elsevier.com
 
 	Partially covered hosts on *elsevier.com:
 
-		- dev ¹
 		- (www.)?developers ²	(^ → www)
 
-	¹ Avoiding MCB
 	² Some paths redirect to http
 
-
-	Fully covered hosts in *elsevier.com:
-
-		- (www.)?
-		- api
-		- help
-		- linkinghub
-		- support	(→ help.elsevier.com)
-
-
-	Insecure cookies are set for these hosts:
-
-		- evolve.elsevier.com
-		- help.elsevier.com
-
-
-	Mixed content:
-
-		- css on dev from getbootstrap.com ¹
-
-		- Images, on:
-
-			- ees.elsevier.com from $self
-			- linkinghub.elsevier.com from www.sciencedirect.com
-			- linkinghub.elsevier.com from www2.us.elsevierhealth.com
-
-	¹ Unsecurable <= refused
-
 -->
 <ruleset name="Elsevier.com (partial)">
 
-	<!--	Direct rewrites:
-				-->
 	<target host="elsevier.com" />
+	<target host="www.elsevier.com" />
 	<target host="acw.elsevier.com" />
 	<target host="api.elsevier.com" />
 	<target host="auth.elsevier.com" />
 	<target host="covers.elsevier.com" />
 	<target host="dev.elsevier.com" />
+	<target host="developers.elsevier.com" />
 	<target host="www.developers.elsevier.com" />
-	<!--target host="ees.elsevier.com" /-->
-	<!--target host="www.ees.elsevier.com" /-->
+		<exclusion pattern="^http://www\.developers\.elsevier\.com/action/" />
+			<test url="http://www.developers.elsevier.com/action/devprojects" />
```

Comment:
Some paths were still redirecting to `http` when I tested it AFAIR but it doesn't seem to be the case anymore.